pimcore/models/Asset/Listing/Dao.php

```diff
@@ -111,6 +111,8 @@ public function getTotalCount()
         $select = $this->getQuery([new Expression('COUNT(*)')]);
         $select->reset(QueryBuilder::LIMIT_COUNT);
         $select->reset(QueryBuilder::LIMIT_OFFSET);
+        $select->reset(QueryBuilder::ORDER);
```

Comment:
how is this related to the whole task ?

Reply:
It was just a test if you review the code :p. 

The line is not really related to this Pull request. I think i had just selected the this branch when i committed the issue - sorry . 
Anyway - this should be merged in the core too.